store/copr/mpp.go

```diff
@@ -227,6 +229,25 @@ func (m *mppIterator) handleDispatchReq(ctx context.Context, bo *tikv.Backoffer,
 	m.establishMPPConns(bo, req, taskMeta)
 }
 
+// TODO: retry once failed?
+func (m *mppIterator) cancelMppTask(bo *tikv.Backoffer, meta *mpp.TaskMeta) {
+	killReq := &mpp.CancelTaskRequest{
+		Meta: meta,
+	}
+
+	wrappedReq := tikvrpc.NewRequest(tikvrpc.CmdMPPCancel, killReq, kvrpcpb.Context{})
+	wrappedReq.StoreTp = kv.TiFlash
+
+	// send cancel cmd to all TiFlash stores
+	for _, addr := range m.store.GetRegionCache().GetTiFlashStoreAddrs() {
+		_, err := m.store.GetTiKVClient().SendRequest(bo.GetCtx(), addr, wrappedReq, tikv.ReadTimeoutUltraLong)
+		logutil.BgLogger().Debug("cancel task ", zap.Uint64("query id ", meta.GetStartTs()), zap.String(" on addr ", addr))
```

Comment:
If you mean to keep this log, It's better to log the task id as well.

Reply:
tiflash only cares about the query id, and then cancel all tasks of the query id. so here print the query id and address.